atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractTableMappingService.java

```diff
@@ -94,31 +94,32 @@ private TableReference getFullTableName(TableReference shortTableName) {
     private final ConcurrentHashMap<TableReference, Boolean> unmappedTables = new ConcurrentHashMap<>();
 
     @Override
-    public Set<TableReference> mapToFullTableNames(Set<TableReference> tableRefs) {
-        Set<TableReference> newSet = Sets.newHashSet();
+    public Map<TableReference, TableReference> generateMapToFullTableNames(Set<TableReference> tableRefs) {
+        Map<TableReference, TableReference> inputNameToFullTableName = Maps.newHashMapWithExpectedSize(
```

Comment:
In the gotham world it we have tables that are unmapped, so technically wasn't sure if that meant they really had a short name, which is why I went with input.  Happy to change input to short if you feel that's more appropriate.